app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,24 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: user_session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br>
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
```

Comment:
as discussed you should be able to easily 🤞 use [govuk-formbuilder helpers](https://govuk-form-builder.netlify.app/form-elements/text-input/) for email and password field styling

Reply:
👍 have added govuk styling